tests/calculations.test.js

```diff
@@ -0,0 +1,42 @@
+let describe = require('mocha').describe
+let it = require('mocha').it
+let expect = require('chai').expect
+let strictEqual = require('../calculation')
+
+
+describe('calculation', () => {
+  describe('strictEqual', () => {
+    it('return true if values are strict equals', () => {
+      const equality = strictEqual(5, 5)
+
+      expect(equality).to.equal(true)
+    })
+    it('return true if values are strict equals', () => {
```

Comment:
Your it blocks should be more descriptive. 
`it('returns true when type and values are the same')`
`it('returns false when the type is the same but values are different')`
etc...